modules/seqexec/web/client/src/main/resources/less/style.less

```diff
@@ -429,32 +432,81 @@ body {
   margin-bottom: 1em;
 }
 
-.SeqexecStyles-acProgressRow {
+.SeqexecStyles-tableDetailRow {
   display: flex;
   flex-grow: 1;
   flex-direction: column;
   width: 100%;
   height: 100%;
-  padding: @table_row_left_padding;
+  padding: @table_detail_row_top_padding @table_detail_row_right_padding @table_detail_row_bottom_padding @table_row_left_padding;
   z-index: 100;
   position: relative;
   margin-left: 20px;
 }
 
+.SeqexecStyles-nodAndShuffleDetailRow {
+  display: flex;
+  justifyContent: spaceBetween;
```

Comment:
I think this should be `space-between`
https://css-tricks.com/almanac/properties/j/justify-content/

Reply:
Yes! And since it doesn't seem to be affecting the display, we should remove it ;)